mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -48,11 +48,17 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 	FIELD field_5975 NAME_VISIBLE Lnet/minecraft/class_2940;
 	FIELD field_5976 horizontalCollision Z
 	FIELD field_5978 ENTITY_ID_COUNTER Ljava/util/concurrent/atomic/AtomicInteger;
+		COMMENT A generator of unique entity {@link #entityId network IDs}. The generated
+		COMMENT ID for client entities are useless and discarded subsequently through
+		COMMENT {@link #setId(int)} calls.
 	FIELD field_5979 passengerList Lcom/google/common/collect/ImmutableList;
 	FIELD field_5981 uuidString Ljava/lang/String;
 	FIELD field_5982 prevYaw F
 	FIELD field_5985 ignoreCameraFrustum Z
 	FIELD field_5986 entityId I
```

Comment:
This should be renamed to match the accessors IMO.

Reply:
Hmm, does it have potential conflicts? Good thing is it's a private field and shouldn't conflict